### PR TITLE
converted Overview from mixer to jekyll

### PR DESCRIPTION
**Pull request by ruthhoffmann**
Overview should be now successfully converted to jekyll.

### CODE REVIEW

File: Overview/index.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Empty document to protect directory
+layout: default
+---
```
Comment (PaulaHaehndel) [inline Overview/index.html]: GitHup complains about "non newline at end of file"

Comment (ruthhoffmann) [reply Overview/index.html]: Added a newline.

File: Overview/Capabilities/permgroups.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+---
+title: Permutation and Matrix Groups
+layout: default
+---
 
 <p>
   For an overview of computational methods for permutation groups see
   the book [<a href="/Doc/references.html#Se03">Se03</a>] of
-  <mixer person="Akos Seress" data="name"/>. Historically, these methods
+  Akos Seress (1958-2013). Historically, these methods
```
Comment (PaulaHaehndel) [inline Overview/Capabilities/permgroups.html]: I think he is written **Á**kos Seress.
Do we want to be consistent with writing years behind names?

Comment (ruthhoffmann) [reply Overview/Capabilities/permgroups.html]: Weirdly, in the database (and on the old page) it was without the accent and for some reason the years were part of his name in there too. Happy to take it or leave it.

Comment (ruthhoffmann) [reply Overview/Capabilities/permgroups.html]: Raised globally in issue #160 . Thanks Paula!

**Pull request merged**